pkg/catalog/ws_http.go

```diff
@@ -54,10 +55,11 @@ func (mc *MeshCatalog) witesandHttpClient() {
 		}
 		// learn remote pods
 		for clusterId, remoteK8s := range wc.ListRemoteK8s() {
-			remotePods, err := mc.QueryRemoteOSM(wc, remoteK8s.OsmIP)
+			remoteEdgePods, err := mc.QueryRemoteEdgePods(wc, remoteK8s.OsmIP)
```

Comment:
why do you want to call them as remoteEdgePods ?
Because, the same code executed both on Master as well as on Satellite.
So, for Master OSM, satellite is the remote and for Satellite OSM, master is the remote.